parkkihubi/settings.py

```diff
@@ -5,6 +5,7 @@
 from django.utils.encoding import smart_str
 from django.utils.translation import gettext, gettext_lazy
 from environ import Env
+import environ
```

Comment:
Please remove the redundant import environ line and instantiate Env() directly. This keeps the imports lean and avoids namespace duplication.